sql/database.sql

```diff
@@ -4826,6 +4832,7 @@ CREATE TABLE IF NOT EXISTS `patient_tracker_element` (
   `status`             varchar(31)  NOT NULL default '',
   `seq`                varchar(4)   NOT NULL default '' COMMENT 'This is a numerical sequence for this pt_tracker_id events',
   `user`               varchar(255) NOT NULL default '' COMMENT 'This is the user that created this element',
+  `reason`             varchar(1000)  NOT NULL default '',
```

Comment:
Change this field to varchar(255) or text. I don't think anyone is going to do 255 characters much less 1000 but text will allow them to write a book.